custom_components/frigate/camera.py

```diff
@@ -167,9 +167,7 @@ def __init__(
                     **self._camera_config
                 )
             else:
-                self._stream_source = (
-                    f"rtsp://{URL(self._url).host}:8554/{self._cam_name}"
-                )
+                self._stream_source = f"rtsp://{URL(self._url).host}:8554/{self._cam_name}?video=copy&audio=aac"
```

Comment:
If this is temporary while we're waiting for another fix, please add a TODO and a reminder to remove it when <link> is  fixed.

Reply:
Good point. Done.